scripts/oneoff/generate-dos-2-master-csv.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Generate a CSV (to stdout) with per-lot draft statistics for each supplier
+
+Usage:
+    scripts/oneoff/generate-dos-2-master-csv.py <base_url> <auth_token>
```

Comment:
Got that from the client base class itself here:
`dmapiclient.base.BaseAPIClient`
But I suppose as an arg name fro a script it doesn't make a great deal of sense to propagate that back up into the command line doco. FIXED 👍